docs/devupdate.md

```diff
@@ -0,0 +1,958 @@
+---
+title: Moodle 4.0 developer update
+tags: []
```

Comment:
Could still benefit from some tags I think here

Reply:
My apologies! I totally forget it!! I've added a couple of tags to this page! :-) Well spotted!